pallets/schema/src/mock.rs

```diff
@@ -120,7 +120,6 @@ pub mod runtime {
 		type DbWeight = RocksDbWeight;
 		type Version = ();
 		type PalletInfo = PalletInfo;
-		type AccountData = pallet_balances::AccountData<Balance>;
```

Comment:
can't remove this completely. use type AccountData = ();

Reply:
Done